docs/develop/integrations/amazon-eventbridge.md

```diff
@@ -0,0 +1,148 @@
+5. The event payload is transformed and calls the Momento HTTP API
+
+Given the event-based nature of this workflow, *it is an asynchronous process* and your code will not wait for completion before continuing. If something goes wrong or the API returns an error, the message will be delivered to a [dead letter queue](https://aws.amazon.com/what-is/dead-letter-queue/).
+
+For the tl;dr, you can deploy the EventBridge resources directly into your account by clicking this button [![Launch stack button](/img/cloudformation-launch-stack.png)](https://console.aws.amazon.com/cloudformation/home#/stacks/create/review?stackName=momento-api&templateURL=https://momento-developers.s3.amazonaws.com/api-destinations.yaml)
+
+## Deploying to AWS
+
+When you hit the *Launch stack* button above, it will open a tab to the CloudFormation console in AWS. The screen will prompt you for your Momento auth token, which can be created via the [Momento Console](https://console.gomomento.com/tokens). You can create a super user token in the region you want to deploy this stack into. Once you generate your token, come back to the console and fill in the property. This will be stored securely in Secrets Manager on your behalf.
+
+You optionally can provide the EventBridge event bus name that will rules will trigger from. It automatically fills in the *default* event bus, but you can use any existing event bus you'd like. The deployment will fail if you do not provide a valid bus name.
+
+![CloudFormation UI with deployment parameters configured](/img/eb-destination-ui.png)
+
+With the two parameters filled out, you can hit *Create stack* and the resources will be deployed automatically for you, optimized for your deployment region. *Wondering what that means?* Momento, like AWS, is region-based, and your auth tokens target a specific region. For the HTTP API, this means you must [hit a different base URL](/develop/api-reference/http-api.md#regions) depending on the region you wish to use. The quick start we've provided will determine the appropriate base URL upon deployment so you don't have to think about it 👍
+
+This also means the integration will only work in the Momento-supported AWS Regions:
+
+* us-east-1
+* us-west-2
+* ap-northeast-1
+* ap-south-1
+* eu-west-1
+
+### Resources
+
+The deployed stack will create the following resources in your AWS account:
+
+* 1x [EventBridge Connection](https://docs.aws.amazon.com/eventbridge/latest/APIReference/API_Connection.html) - for storing connection info to Momento
+* 3x EventBridge API Destinations - *Cache Item Set*, *Cache Item Delete*, *Topic Publish*
+* 3x EventBridge Rules - for triggering the API calls
+* 1x IAM Role - to allow EventBridge to invoke the destinations when the rules are triggered
+* 1x SQS Queue - for Dead Letter Queue on failed delivery
+
+If you wish to see the raw CloudFormation template, [click here](https://momento-developers.s3.amazonaws.com/api-destinations.yaml).
```

Comment:
just out of curiosity, where are you housing the source for this? It would be good to have an internal breadcrumb trail in case users are inquiring about it and someone else besides you needs to debug.